scripts/deploy/deployDCLControllerV2.js

```diff
@@ -0,0 +1,37 @@
+import ethProvider from 'eth-provider'
+import hre, { web3 } from 'hardhat'
+
+async function main() {
+  let chain
+  let mana
+  let dclRegistry
+  let feeCollector
+
+  switch (process.env.NETWORK) {
+    case 'GOERLI':
+      chain = 5
+      mana = '0xe7fDae84ACaba2A5Ba817B6E6D8A2d415DBFEdbe'
+      dclRegistry = '0x6b8da2752827cf926215b43bb8E46Fd7b9dDac35'
+      feeCollector = '0xE3336140Edfe740a26F9f1912fD52891e6Dd8A35'
+      break
+    default:
+      throw new Error('Invalid network')
+  }
+
+  const provider = ethProvider()
+
+  provider.setChain(chain)
```

Comment:
The newest version of frame allows being connected to multiple chains at the same time. This is required to indicate on which chain the contract is to be deployed to.